apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -679,6 +679,7 @@ type ApprovalRequestObjList interface {
 //   - `TargetUpdateRun`: Points to the cluster staged update run that this approval request is for.
 //   - `TargetStage`: The name of the stage that this approval request is for.
 //   - `IsLatestUpdateRunApproval`: Indicates whether this approval request is the latest one related to this update run.
+//   - `TargetTaskTypeLabel`: Indicates whether this approval request is for the before or after stage task.
```

Comment:
This should still be taskType, it's not a target

Reply:
In case of targetStage, targetUpdateRun we target a specific entity/object but for this case we are describing a characteristic/trait of the StageTask wherther it's Before/After a particular Stage. I'd stick with taskType